test/framework/src/main/java/org/elasticsearch/search/aggregations/AggregatorTestCase.java

```diff
@@ -295,10 +293,6 @@ public boolean shouldCache(Query query) {
         MapperService mapperService = mapperServiceMock();
         when(mapperService.getIndexSettings()).thenReturn(indexSettings);
         when(mapperService.hasNested()).thenReturn(false);
-        DocumentMapper mapper = mock(DocumentMapper.class);
-        when(mapper.typeText()).thenReturn(new Text(TYPE_NAME));
-        when(mapper.type()).thenReturn(TYPE_NAME);
-        when(mapperService.documentMapper()).thenReturn(mapper);
```

Comment:
I wasted several hours trying to work out why my mocked DocumentMapper class wasn't returning the DocumentFieldMapper instance I was asking it to.  This appears to be unnecessary, so I've removed it